### PR TITLE
doc improvements

### PR DESCRIPTION
**Pull request by rolson**
Clarify what the behavior of a force-quit.

### CODE REVIEW

File: Sources/ArcGISToolkit/Components/JobManager/JobManager.swift [hunk 1]
```diff
@@ -44,11 +44,11 @@ import UIKit
 /// serialized when an app is backgrounded so that if the app is terminated the jobs can be
 /// rehydrated upon relaunch of the app.
 ///
-/// Also in iOS if the user of an app removes the app from the app switcher (swiping up) then the
-/// system interprets this as a strong indication that the user does not want the app running.
-/// The consequences of this are two-fold for jobs. One, any background fetch tasks are not given
-/// any time until the app is relaunched again. And two, any background downloads that are in
-/// progress are canceled by the operating system.
+/// Also, in iOS, if the user of an app removes the app from the multitasking UI (aka force quit it),
```
Comment (dfeinzimer) [inline Sources/ArcGISToolkit/Components/JobManager/JobManager.swift:47]: Should it be "quits" instead?

> aka force quits it

Comment (rolson) [inline Sources/ArcGISToolkit/Components/JobManager/JobManager.swift:47]:
```suggestion
/// Also, in iOS, if the user of an app removes the app from the multitasking UI (aka force quits it),
```

**Review by dfeinzimer — APPROVED** [review]
Looks good! 

I see this targets main.. I just want to make sure that's intended since we rarely merge directly into main.

**Comment by rolson** [comment]
no, it should target v.next. Switched, please re-approve.

File: Examples/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 This is an application that shows how to use toolkit components to simplify development of your iOS apps built with the ArcGIS Maps SDK for Swift.
 
-Documentation for the toolkit components is [here](../Documentation/README.md).
+Documentation for the toolkit components is [here](https://developers.arcgis.com/swift/toolkit-api-reference/documentation/arcgistoolkit/).
```
Comment (rolson) [inline Examples/README.md:5]: Looks like if I target v.next I get this commit as well.

**Comment by rolson** [comment]
Closing, in-lieu of this [other PR](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/pull/800). 

BTW, It looks like main needs to be merged *back* into v.next.

**Comment by dfeinzimer** [comment]
> BTW, It looks like main needs to be merged _back_ into v.next.

Screenshot 2024-07-10 at 10 28 54 AM

Hmm for some reason GitHub doesn't seem to want to allow that - disables the button.

**Comment by dfeinzimer** [comment]
Actually, never mind, I didn't need a PR to do it. Done now, thanks!

**Pull request closed without merging**